jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -52,6 +52,59 @@ void init() throws SQLException {
     dbMetaData = new TimestreamDatabaseMetaData(mockConnection);
   }
 
+  /**
+   * Checks that an empty result set is returned for getCatalogs
+   */
+  @Test
+  void testGetCatalogsWithResult() throws SQLException {
+    initializeWithResult();
+    try (ResultSet resultSet = dbMetaData
+            .getCatalogs()) {
+      Assertions.assertFalse(resultSet.next());
+    }
+  }
+
+  /**
+   * Checks that a result set containing database name "testDB" is returned for getSchemas with no parameters
+   */
+  @Test
+  void testGetSchemasWithResult() throws SQLException {
+    initializeWithResult();
+
+    try (ResultSet resultSet = dbMetaData
+            .getSchemas()) {
+      testGetSchemasResult(resultSet);
+    }
+  }
+
+  /**
+   * Checks that a result set containing database name "testDB" is returned for getSchemas with null parameters
+   */
+  @Test
+  void testGetSchemasNullParamWithResult() throws SQLException {
+    initializeWithResult();
+
+    try (ResultSet resultSet = dbMetaData
+            .getSchemas(null, null)) {
+      testGetSchemasResult(resultSet);
+    }
+  }
+
+  /**
+   * Checks that a result set containing database name "testDB" is returned for getSchemas with schemaPattern
+   * @param schemaPattern Schema pattern to be tested
+   * Input values tested for schemaPattern: {"%", "testDB", "%testDB%"}
+   */
+  @ParameterizedTest
+  @ValueSource(strings = {"%", "testDB", "%testDB%"})
```

Comment:
it would be good to add negative test cases with invalid schema pattern that is expected to fail and the test can ensure that the right exception is received from JDBC driver

Reply:
The invalid schema pattern test is added.